src/Storages/System/StorageSystemPartsBase.cpp

```diff
@@ -138,7 +138,7 @@ StoragesInfoStream::StoragesInfoStream(const SelectQueryInfo & query_info, Conte
 
                     String engine_name = storage->getName();
                     UUID storage_uuid = storage->getStorageID().uuid;
-                    if (database->getEngineName() == "Ordinary")
+                    if (database->getEngineName() != "Atomic")
```

Comment:
There is also Replicated database engine.,
as well as Overlay, which is used by clickhouse-local.

Reply:
This looks like the wrong way to solve the problem. It should be part of IDatabase interface.

Reply:
Indeed. What do you think of just checking UUID instead of expanding IDatabase?

Reply:
Yes, it will be better.